emu/src/mem.rs

```diff
@@ -51,16 +51,17 @@ impl Mem {
     }
     pub fn write_u8(&mut self, address: u32, value: u32) {
         match address & 0xFFFFFF {
-            0..=0x3FFFFF => {},
+            0..=0x9FFFFF => {},
             0xA00000..=0xA03FFF => self.z80.write_byte(address, value),
             0xA04000..=0xA0FFFF => {},
-            0xA10000..=0xA1001F => self.io.write_byte(address, value),
-            0xA11100..=0xA112FF => self.z80.ctrl_write(address, value),
+            0xA10000..=0xA10FFF => self.io.write_byte(address & 0xFFF01F, value),
+            0xA11000..=0xA110FF => {},
```

Comment:
this is part of the IO still so should be handled there instead (in io.write_byte)

Reply:
Well, the thing is that 0xA10000 is mirrored through at least 0xA100FF, so I don't know how exactly to handle that without more extensive changes to the code.

Reply:
you could just do it without mirroring. do the same as here (nothing) but within that function instead

Reply:
I guess, yeah